src/bytecode.h

```diff
@@ -0,0 +1,78 @@
+#ifndef _KANSAI_KBC_
+#define _KANSAI_KBC_
+
+#include <stdint.h>
+
+#define MAX_OPCODE 10
+#define DEFAULT_DOT_ENTRIES 100
+#define DEFAULT_OPTIONS 4 /* I find it very unlikely we'll see more than 4. */
```

Comment:
idk about this, I've seen VN and non-VN games with a larger amount of options (for things like moving location, selecting items, not necessarily dialogue choices)

Reply:
except that would be a custom UI, not an option selection screen.